### ISSUE TITLE
CMD + M overrides Minimize action

### ISSUE THREAD

**Issue by pikor** [ProxymanApp/Proxyman#705]
### Proxyman version?
Version 2.13.0 (21300)

### macOS Version? (Ex. mac 10.14)
10.15.7

### Steps to reproduce 
1. Open Proxyman
2. Hit CMD+M
3. Depending on the selection, either nothing happens or "Add comment" windows opens up

### Expected behavior
1. Standard macOS behavior (the app is minimized).

When I switched from Charles to Proxyman I was delighted that a) Proxyman is a native app, b) macOS default shortcuts (like minimize) work as they should. However, some recent version of Proxyman overrides Minimize action (CMD+M). I [added custom macOS app shortcut](https://support.apple.com/guide/mac-help/create-keyboard-shortcuts-for-apps-mchlp2271/mac) to workaround the issue, but with no luck.

I understand that the Comments feature is a great Proxyman addition, but at the same time, standard macOS shortcuts should not be overridden. Thanks to that, it's easier to work on the platform.

**Comment by NghiaTranUIT** [comment]
Hi,

Sorry for this inconvenience. It's my mistake to accidentally override CMD+M to add the comment.

I'm fixing it and send you a beta build now 👍

**Comment by pikor** [comment]
Wow, @NghiaTranUIT thank you for such quick response! 

It's great to hear that CMD+M is coming back, thanks! :)

**Comment by NghiaTranUIT** [comment]
yes, it should be back 😄 Ideally, Proxyman should not override any system keyboard shortctut 🙌

**Comment by NghiaTranUIT** [comment]
Hey @pikor let check out this beta build: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_2.13.0_Change_Add_Comment_Shortcut.dmg

I changed the "Add Comment" from CMD+M to CMD+L, so it doesn't conflict with the Minimize Window shortcut 👍 

Thank you for reporting this issue 🎉 🌮

**Comment by pikor** [comment]
Looks good, thanks! 🙌
